src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("./fonts/Roboto-Medium.ttf") format("truetype");
+  font-weight: 500;
+  font-style: normal;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  min-width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav {
+  justify-content: flex-end;
+  display: block;
+}
+
+.nav__table {
+  list-style-type: none;
+  display: flex;
+  margin: 23px 0 22px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  padding: 23px 0 22px;
```

Comment:
I believe you don't need this padding. Try to delete it, layout should not be changes

Reply:
You now, its 100% true, but fot "npm start", but for "npm test" happend something bad. Look.
![image](https://user-images.githubusercontent.com/78905554/219295205-3d7cff29-156a-464d-86f2-a39c258be427.png)
![image](https://user-images.githubusercontent.com/78905554/219295250-b83020d0-86c6-48ef-8584-dab5b675ef86.png)